script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -1106,7 +1106,7 @@ The Executor's sword repels it.
 [sha:ad9c3b95bdf10fb9e4a8dd008a343bc6f050773a]{
 -- Page 642, Offset 12556.
 -- 人智を超越した斬り合いはいつ果てるともなく続いていく。
-The sword fight that has long transcended human comprehension shows no sign of stopping.
+The swordfight that has long transcended human comprehension shows no sign of stopping.
```

Comment:
"sword fight" is infinitely more common. I do not believe the google dictionary here